commands/hrandmember.md

```diff
@@ -0,0 +1,38 @@
+When called with just the `key` argument, return a random field from the hash value stored at `key`.
+
+If the provided `count` argument is positive, return an array of `count` **distinct fields**.
```

Comment:
i know you provided that info at the bottom, but i feel it should be mentioned here too, specifically because the current wording is really incorrect (suggesting that exactly count is returned).
```suggestion
If the provided `count` argument is positive, return an array of **distinct fields** either the size of the provided `count` or the number of elements in the hash, which ever is lower).
```

Reply:
Resolved via ca2bd9a3da2ad5e88dd2ef434cf21e6062b8649f